src/app/screens/Calculator/index.js

```diff
@@ -0,0 +1,50 @@
+import React, {useState} from 'react';
+import {View, TextInput} from 'react-native';
+import {KeyboardAwareScrollView} from 'react-native-keyboard-aware-scroll-view';
+
+import styles from './styles';
+import Button from './components/Button';
+import { retrieveButtons } from './utils';
+
+const Calculator = () => {
+  //Variables y funcionesa
+  const [display, setDisplay] = useState('');
+
+  const leerPresionado = target => {
+    GET_BUTTONS.find(button => button.label === target).action();
+  };
```

Comment:
You are mixing things up. You should have each function separated from the buttons, not hardcoded in them. Then use those in the onKeyPress, and then map each button with each function.

So you can have an object which keys and then use them by key instead of using find everytime.

Reply:
leerPresionado should be in english

Reply:
This function was used to find the action of the key pressed from the native keyboard.

I moved it into the textinput, tell me if it's okay.